clab/clab_test.go

```diff
@@ -22,58 +21,6 @@ import (
 	"golang.org/x/exp/slices"
 )
 
-func Test_createNamespaceSharingDependencyOne(t *testing.T) {
-	mockCtrl := gomock.NewController(t)
-	defer mockCtrl.Finish()
```

Comment:
why this was removed?

Reply:
These are tests for the DependencyManager. But the DM does not have these functions any more.

Reply:
dm.AddDependency is gone ... dependencies are added via the nodes directly